pkgs/development/python-modules/llm-claude-3/default.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  anthropic,
+  llm,
+  pytestCheckHook,
+  pytest,
+  pytest-recording,
+  nix-update-script,
+  ...
+}:
+buildPythonPackage rec {
+  pname = "llm-claude-3";
+  version = "0.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "simonw";
+    repo = pname;
+    rev = "refs/tags/${version}";
+    hash = "sha256-5qF5BK319PNzB4XsLdYvtyq/SxBDdHJ9IoKWEnvNRp4=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+  buildInputs = [ llm ];
+  propagatedBuildInputs = [ anthropic ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+  # Test suite requires network access to talk to Claude (duh).
+  doCheck = false;
+
+  pythonImportsCheck = [ "llm_claude_3" ];
+
+  passthru = {
+    optional-dependencies = {
+      test = [
+        pytest
+        pytest-recording
+      ];
+    };
+    updateScript = nix-update-script { };
+  };
+
+  meta = with lib; {
+    description = "LLM plugin for interacting with the Claude 3 family of models";
+    homepage = "https://github.com/simonw/llm-claude-3";
+    license = licenses.asl20;
+    maintainers = builtins.attrValues { inherit (maintainers) jkachmar; };
```

Comment:
```suggestion
    maintainers = with lib.maintainers; [ jkachmar ];
```

Reply:
why? i've generally tended away from `with` (i missed the earlier `with` in a comment you made above so i'll fix that)